pkg/kubectl/cmd/util/helpers.go

```diff
@@ -46,6 +46,7 @@ import (
 
 const (
 	ApplyAnnotationsFlag = "save-config"
+	AddOnLabel           = "kubernetes.io/cluster-service"
```

Comment:
this should be more generic if the behavior it is controlling is write protection, e.g. "kubernetes.io/write-protect=true"